stable/guardian/values.yaml

```diff
@@ -0,0 +1,9 @@
+app:
+  image:
+    repository: odpf/guardian
+    pullPolicy: Always
+    tag: latest
+  container:
+    entryPoint:
+    - serve
+  config: {}
```

Comment:
This should include guardian specific configs instead of an open map

Reply:
Also, could we add ingress config, secret configs for guardian here?